assets/helpers/page/page.js

```diff
@@ -116,14 +124,29 @@ function statelessInit() {
   const country: IsoCountry = detectCountry();
   const participations: Participations = abTest.init(country);
   analyticsInitialisation(participations);
+}
+
+// Enables redux devtools extension and optional redux-thunk.
+/* eslint-disable no-underscore-dangle */
+function storeEnhancer<S, A>(thunk: boolean): StoreEnhancer<S, A> | typeof undefined {
```

Comment:
What is `type of undefined` for here? Could you put  `? StoreEnhancer<S, A>`  instead? or `StoreEnhancer<S, A> | 'undefined'`

Reply:
The result of `typeof undefined` is the string 'undefined', I thought the second option would be equivalent and more readable.

Reply:
It's not a JavaScript `typeof`, it's a [Flow `typeof`](https://flow.org/en/docs/types/typeof/).

Reply:
As discussed offline, @svillafe is correct, this makes no sense, because, for example:

```js
type Foo = StoreEnhancer<S, A> | null;
```

is valid 🤔.